frontend/src/employee-frontend/components/reports/VoucherServiceProviders.tsx

```diff
@@ -152,12 +152,7 @@ export default React.memo(function VoucherServiceProviders() {
           childCount: childCount,
           sum: formatCents(monthlyPaymentSum, true)
         }))
-        .sort((l, r) =>
-          `${l.areaName}-${l.unitName}`.localeCompare(
-            `${r.areaName}-${r.unitName}`,
-            'fi'
-          )
-        )
+        .sort((l, r) => `${l.unitName}`.localeCompare(`${r.unitName}`, 'fi'))
```

Comment:
``  `${l.unitName}`.localeCompare(`${r.unitName}`, 'fi') `` can be simplified to `l.unitName.localeCompare(r.unitName, 'fi')`.